genrl/deep/common/policies.py

```diff
@@ -73,7 +75,16 @@ def __init__(
         )
         self.action_dim = action_dim
 
-        self.conv, output_size = cnn((framestack, 16, 32))
+        if "channels" in kwargs:
+            channels = [framestack]
+            for i in range(len(kwargs["channels"])):
+                channels.append(kwargs["channels"][i])
+            channels = tuple(channels)
+
+        else:
+            channels = (framestack, 16, 32)
+
```

Comment:
Maybe you could shift this to the `cnn` function in `utils.py` itself? Also, "channels" may not necessarily be in `kwargs`.

Reply:
I guess I'll just remove this, this exists in the `cnn` function in `utils.py` alrready